src/services/audius-backend/Rewards.ts

```diff
@@ -0,0 +1,149 @@
+import { Name } from 'common/models/Analytics'
+import { Level } from 'common/store/errors/level'
+import { reportToSentry } from 'common/store/errors/reportToSentry'
+import { track } from 'store/analytics/providers'
+import { isMobile, isElectron } from 'utils/clientUtil'
+
+/**
+ * Reports Rewards claim outcome to Identity, Amplitude, Sentry
+ * with a fire-and-forget approach
+ */
+
+export class ClientRewardsReporter {
+  source: 'mobile' | 'electron' | 'web'
+  libs: any
+
+  constructor(libs: any) {
+    this.source = isMobile() ? 'mobile' : isElectron() ? 'electron' : 'web'
+    this.libs = libs
+  }
+
+  reportSuccess({
+    userId,
+    challengeId,
+    amount,
+    specifier
+  }: {
+    userId: string
+    challengeId: string
+    amount: number
+    specifier: string
+  }) {
+    ;(async () => {
+      try {
+        await track(Name.REWARDS_CLAIM_SUCCESS, {
```

Comment:
not sure the benefit of awaiting here at all. since it's fire & forget, you probably don't want to fail to call `sendAttestationResult` if the `track` call itself fails.

since maybe you want the log, maybe this should be two async methods? no need to change, just musing

Reply:
Yeah I think you're right - no need for async will remove

Reply:
JK we need it, otherwise the try catch doesn't catch the exceptions

Reply:
My point here was more that
```
try {
  await track
  await send
}
```

will lead to `send` never getting called if `track` throws
so probably better to do

```
try {
  await track
} catch () {}
try {
  await send
} catch () {}

```

or wrapping them in each in the async call to not do serially

```
async () => {
  try {
    await track
  } catch () { }
}()

async () => {
  try {
     await send
  } catch () { }
}()
```